packages/node-app/src/controllers/giveaway.controller.ts

```diff
@@ -155,6 +172,18 @@ export const updateGiveaway = async (req: Request, res: Response) => {
     const giveaway = await Giveaway.findByIdAndUpdate(id, updateFields, {
       new: true,
     });
+    await GiveawayLog.create({
+      action: 'update-giveaway',
+      author: req.user.email,
+      giveaway_id: giveaway.id,
+      changes: {
+        title,
+        description,
+        prize,
+        rules,
+        image,
+      },
```

Comment:
Maybe for the update we should only log what changed, just thinking that we will save the image several times with no need for example